README.md

```diff
@@ -29,14 +29,65 @@ available for telescope (as native component or as lua component).
 
 ## Installation
 
-To get **fzf-native** working, you need to build it with either `cmake` or `make`. As of now, we do not ship binaries.
-Both install methods will be supported going forward.
+`telescope-fzf-native` is mostly a native binary. We do not commit this to the
```

Comment:
maybe add another sentence on top, that there are two versions to get the binary, either building it yourself or downloading the latest ci build. Because i would still suggest people to build the binary themselves if they are on a linux system.